docs/design/offline.md

```diff
@@ -123,13 +123,13 @@ It's okay if the app selects multiple streams of a type.  For text, this may be
 necessary, for example, to store subtitles for all languages.  For audio it may
 be necessary as well, for similar reasons.  For video it makes little sense, but
 we won't restrict it.  Apps can make whatever decision they want about tracks to
-store, but we should document best practice.
+store, but we should document best practices.
 
-We could also issue a warning if multiple tracks of same type, kind, and
-language are selected, since offline adaptation is currently an uncommon
+We could also issue a warning if multiple tracks of the same type, kind, and
+language is selected since offline adaptation is currently an uncommon
 scenario and may indicate a mistake on the part of the app developer.  For
-example, this would trigger on two video with different bitrate or size, or two
-audio of same language but different bitrate, or two text of same language and
+example, this would trigger on two video with different bitrate or sizes, or two
```

Comment:
"Two video" should say "two videos" (plural), and "bitrate" should also be pluralized.